content/v2.0/telegraf-configs/update.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Update a Telegraf configuration
+description: >
+  Use the InfluxDB user interface (UI) and the [`influx` CLI](/v2.0/reference/cli/influx/)
+  to update InfluxDB Telegraf configurations.
+weight: 103
+menu:
+  v2_0:
+    name: Update a config
+    parent: Telegraf configurations
+aliases:
+  - /v2.0/write-data/no-code/use-telegraf/auto-config/update-telegraf-config/
+  - /v2.0/collect-data/use-telegraf/auto-config/update-telegraf-config
+---
+
+Use the InfluxDB user interface (UI) and the [`influx` CLI](/v2.0/reference/cli/influx/)
+to update InfluxDB Telegraf configurations.
+
+- [Use the InfluxDB UI](#use-the-influxdb-ui)
+- [Use the `influx` CLI](#use-the-influx-cli)
+- [Customize an existing Telegraf configuration](#customize-an-existing-telegraf-configuration)
+
+## Use the InfluxDB UI
+The InfluxDB UI allows you to update the **name** or **description** of a Telegraf configuration.
+
+{{% note %}}
+You cannot modify Telegraf settings in existing Telegraf configurations directly
+through the InfluxDB UI.
+[Update a Telegraf configuration with the influx CLI](#use-the-influx-cli)
+to modify existing Telegraf configuration settings.
+{{% /note %}}
+
+1. In the navigation menu on the left, select **Data** (**Load Data**) > **Telegraf**.
+
+    {{< nav-icon "load data" >}}
+
+2. Hover over the configuration you want to edit and click **{{< icon "pencil" >}}**
+   to update the name or description.
+3. Press Return or click out of the editable field to save your changes.
+
+## Use the influx CLI
+Use the [`influx telegrafs update` command](/v2.0/reference/cli/influx/telegrafs/update/)
+to update an existing InfluxDB Telegraf configuration name, description, or settings
+from a Telegraf configuration file on your local machine.
+
+Provide the following:
+
+- **Telegraf configuration ID** (shown in the output of `influx telegrafs`)
+- **Telegraf configuration name**
+- **Telegraf configuration description**
+- **Telegraf configuration file**
+
+{{% warn %}}
+If a **name** and **description** are not provided, they are replaced with empty strings.
+{{% /warn %}}
```

Comment:
note or warning? (was note above)

Reply:
Warning here, because existing name and description will be replaced. Note above, because the configuration is new so nothing is overwritten.